### PR TITLE
Minor UI changes

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys hvae a look. Changes include:
- Improving position of switch teams in the admin console and adding an icon for that option.
- Adding "Switch to " text for teams.
- Enabling time stamp for all posts on mobile.
- Fixing thumbnail width of files in compact view.

### CODE REVIEW

File: webapp/components/admin_console/admin_navbar_dropdown.jsx [hunk 1]
```diff
@@ -99,20 +103,21 @@ export default class AdminNavbarDropdown extends React.Component {
...
                         <li>
                             <Link
                                 to={'/select_team'}
                             >
+                                <i className='fa fa-exchange'/>
                                 <FormattedMessage
                                     id='admin.nav.switch'
-                                    defaultMessage='Switch to {display_name}'
+                                    defaultMessage='Switch Teams'
                                     values={{
```
Comment (enahum) [inline webapp/components/admin_console/admin_navbar_dropdown.jsx]: If you removed the need for a value, then why not delete the `values` attr, it seems to me that is no longer needed.

**Comment by asaadmahmoodspin** [comment]
Removed it @enahum.

**Comment by coreyhulen** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**